### PR TITLE
feat: gated fsync kernel (matches coreos-stable)

### PR DESCRIPTION
**Pull request by m2Giles**
## Thank you for contributing to the Universal Blue project!

Please [read the Contributor's Guide](https://universal-blue.org/contributing.html) before submitting a pull request.

### CODE REVIEW

File: .github/workflows/reusable-build.yml [hunk 1]
```diff
@@ -77,7 +80,11 @@ jobs:
           export BUILDER_IMAGE=quay.io/fedora/fedora
           echo "BUILDER_IMAGE=${BUILDER_IMAGE}" >> $GITHUB_ENV
           echo "FQ_BUILDER_IMAGE=${BUILDER_IMAGE}:${{ matrix.fedora_version }}" >> $GITHUB_ENV
-          export KERNEL_IMAGE=${{ matrix.kernel_flavor }}-kernel
+          if [[ "${{matrix.kernel_flavor}}" != "fsync-gated" ]]; then
```
[the remark this answers is not in the document]
Comment (m2Giles) [reply .github/workflows/reusable-build.yml:83]: Only reason was that's just how I was thinking. I agree that not using the negative is better

**Review by bsherman — COMMENTED** [review]
My only comment is I struggle with the name 'fsync-gated' ... it does signify a delayed kernel release, but it doesn't connect much to the fact that we're matching the coreos-stable kernel version.

The thoughts I've had seem either too wordy or suffer from the same problem but just disconnecting another concept.

- `fsync-coreos-stable`
- `fsync-coreos`
- `fsync-stable`
???

Thoughts on another name, or just leave it as is.

**Comment by bsherman** [comment]
@m2Giles there was a comment made in Discord about the `coreos-testing, zfs` failure on fedora 40 builds failing... Just calling it out here... the fsync patches themselves don't seem to be conflicting with the zfs kmod build in 6.9. But 6.10 is failing the zfs kmod build even without fsync patches.

**Comment by m2Giles** [comment]
For name I think fsync-coreos should get the idea across. fsync-coreos-stable would be clearer but unsure if we need that distinction

**Review by bsherman — APPROVED** [review]

**Review by castrojo — APPROVED** [review]
